### PR TITLE
updated ego docs with information on adding other services

### PR DESCRIPTION
**Pull request by MitchellShiell**
As discussed and outlined in title, no rush on this one I kept the DMS-UI naming because I want to address that in it's own PR

### CODE REVIEW

File: markdown/documentation/ego/installation/adding-overture-services.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+| Status             | Approved                                    |
+| Application Type   | CLIENT                                      |
+| Client             | DMS-UI                                      |
+| Client Secret      | DMS-UI                                      |
+| Redirect URI       | http://localhost:3000/api/auth/callback/ego |
+| Error Redirect URI | http://localhost:3000/error                 |
+
+You can leave the Groups and Users fields blank. Click the **save** button on the top right of the panel.
+
+2.  **Create an environment variable file:** Within the cloned DMS-UI repository locate your `.env.dmsui` file, if you have not created one yet duplicate the `env.schema` file and rename it to `.env.dmsui`.
```
Comment (leoraba) [inline markdown/documentation/ego/installation/adding-overture-services.md:27]: Probably good to clarify first this step is on DMS-UI side, modify the following environment variables that are required from DMS-UI to work with EGO.

**Review by leoraba — APPROVED** [review]
👍

File: markdown/documentation/ego/installation/adding-overture-services.md (hunk 1 above)
Comment (justincorrigible) [reply markdown/documentation/ego/installation/adding-overture-services.md:27]: this was a timely comment, and I'm glad it popped up in my email.
working on setting things up for the overture demo environment, and I have a hunch one of these is what I'm missing to make it work. thanks!

Comment (MitchellShiell) [reply markdown/documentation/ego/installation/adding-overture-services.md:27]: Will update this on DMS-UI branch (rather merge this with approval for now)

**Pull request merged**